include/swift/AST/DiagnosticsModuleDiffer.def

```diff
@@ -58,6 +58,10 @@ ERROR(decl_added,none,"%0 is added to a non-resilient type", (StringRef))
 
 ERROR(default_arg_removed,none,"%0 has removed default argument from %1", (StringRef, StringRef))
 
+ERROR(conformance_removed,none,"%0 has removed conformance to %1", (StringRef, StringRef))
+
+ERROR(conformance_added,none,"%0 has added conformance to %1", (StringRef, StringRef))
```

Comment:
Adding a conformance is not an API or ABI break.

Also I'm assuming this applies to resilient types too, and if the conformance is defined in an extension?

Reply:
The check performs on all types and if the conformance is defined in extensions. Adding conformances can be a problem for protocols (i.e. adding inherited protools).

Reply:
Please don't model inherited protocols as conformances. Adding an inherited protocol is indeed an API and ABI break, but that's the only case.

A bona-fide "conformance" always involves a concrete type and is its own thing.

Reply:
It's already been scoped to protocol only in TOT and we've reworded the diagnostic message.